sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1614,7 +1637,7 @@ def task_stop(self):
         self.task_stopping_event.set()
         self.task_thread.join()
 
-    def on_port_config_change(self, port_change_event):
+    def on_port_config_change(self, stopping_event, port_change_event):
```

Comment:
where is 'stopping_even' used in this function? can you point me to the caller of this function where its passing 'stopping_even' argument?

Reply:
Hi Prince - thank you for taking a look. On line 1756, 
"port_change_event_handler = functools.partial(self.on_port_config_change, stopping_event)"

adds an argument, stopping_event, to this handler function. when port_change_event_handler gets called in read_port_config_change (xcvrd_utilities/port_mapping.py), it actually calls self.on_port_config_change(stopping_event,port_change_event).

Reply:
I guess I know why you are confused about my change. I just notice that I did not patch the code correctly. You can check my latest diff.